docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -75,35 +75,35 @@ kind: VSphereDatacenterConfig
 metadata:
    name: my-cluster-datacenter
 spec:
-  datacenter: ""
-  disableCSI: false
-  server: ""
-  network: ""
-  insecure: false
-  thumbprint: ""
+  datacenter: ""                     <a href="#datacenter-required"># vSphere datacenter name on which to deploy EKS Anywhere (required) </a>
+  disableCSI: false                  <a href="#disablecsi-optional"># Set to true to not have EKS Anywhere install and manage vSphere CSI driver</a>
+  server: ""                         <a href="#server-required"># FQDN or IP address of vCenter server (required) </a>
+  network: ""                        <a href="#network-required"># Path to the VM network on which to deploy EKS Anywhere (required) </a>
+  insecure: false                    <a href="#insecure-optional"># Set to true if vCenter does not have a valid certificate </a>
+  thumbprint: ""                     <a href="#thumbprint-required-if-insecurefalse"># SHA1 thumprint of vCenter server certificate (required if insecure=false)</a>
 
 ---
 apiVersion: anywhere.eks.amazonaws.com/v1alpha1
 kind: VSphereMachineConfig
 metadata:
    name: my-cluster-machines
 spec:
-  diskGiB:
-  datastore: ""
-  folder: ""
-  numCPUs:
-  memoryMiB:
-  osFamily: ""
-  resourcePool: ""
-  storagePolicyName: ""
-  template: ""
-  users:
-  - name: ""
-    sshAuthorizedKeys:
+  diskGiB:                          <a href="#diskgib-optional"># Size of disk on VMs, if no snapshots</a>
```

Comment:
We updated EKS-A recently to make the disk size configurable regardless of snapshots.
```suggestion
  diskGiB:                          <a href="#diskgib-optional"># Size of disk on VMs</a>
```
Also, if you don't mind, can you remove that from [here](https://anywhere.eks.amazonaws.com/docs/reference/clusterspec/vsphere/#diskgib-optional)?

Reply:
I'll do that in a separate PR, since this already merged.